tests/Kconfig

```diff
@@ -8,6 +8,7 @@ menu "Test"
 
 config ZTEST_MULTICORE_DEFAULT_SETTINGS
```

Comment:
what does this kconfig even do? Multi-image settings are done by sysbuild which doesn't even load this file

Reply:
@nordicjm , what is your suggestion on how to proceed with this PR? Decline it and keep it as is?

Reply:
Yes, it is being removed anyway when https://github.com/nrfconnect/sdk-nrf/pull/19283/ gets merged